diff --git a/third_party/webrtc/modules/audio_processing/aec3/aec_state.cc b/third_party/webrtc/modules/audio_processing/aec3/aec_state.cc
index ccb8639..3a13e6f 100644
--- a/third_party/webrtc/modules/audio_processing/aec3/aec_state.cc
+++ b/third_party/webrtc/modules/audio_processing/aec3/aec_state.cc
@@ -266,14 +266,14 @@ void AecState::UpdateReverb(
   data_dumper_->DumpRaw("aec3_tail_power", tail_power);
 }
 
+#define SABS(_x) (_x >= 0 ? _x : (-1) * _x)
+
 void AecState::EchoAudibility::Update(rtc::ArrayView<const float> x,
                                       const std::array<float, kBlockSize>& s) {
   auto result_x = std::minmax_element(x.begin(), x.end());
   auto result_s = std::minmax_element(s.begin(), s.end());
-  const float x_abs =
-      std::max(std::abs(*result_x.first), std::abs(*result_x.second));
-  const float s_abs =
-      std::max(std::abs(*result_s.first), std::abs(*result_s.second));
+  const float x_abs = std::max(SABS(*result_x.first), SABS(*result_x.second));
+  const float s_abs = std::max(SABS(*result_s.first), SABS(*result_s.second));
 
   if (x_abs < 5.f) {
     ++low_farend_counter_;
@@ -294,7 +294,7 @@ void AecState::EchoAudibility::Update(rtc::ArrayView<const float> x,
 void AecState::EchoAudibility::UpdateWithOutput(rtc::ArrayView<const float> e) {
   const float e_max = *std::max_element(e.begin(), e.end());
   const float e_min = *std::min_element(e.begin(), e.end());
-  const float e_abs = std::max(std::abs(e_max), std::abs(e_min));
+  const float e_abs = std::max(SABS(e_max), SABS(e_min));
 
   if (max_nearend_ < e_abs) {
     max_nearend_ = e_abs;
